css/style.css

```diff
@@ -50,7 +51,7 @@
 }
 
 .childTheme .entry-content {
-	padding-left: 1rem;
+	padding-left: 2rem;
```

Comment:
This should probably switch over to pixels and match the left padding on the other layout elements since the rem size is a bit different. Can you switch this to: `padding-left: 27.5px;`

Reply:
thanks for the details on the style changes!